pkgs/development/python-modules/dbus/default.nix

```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, fetchPypi, buildPythonPackage, python, pkg-config, dbus, dbus-glib, isPyPy
-, ncurses, pygobject3, isPy3k }:
+, ncurses, pygobject3, isPy3k, python3 }:
```

Comment:
Don't pass in `python3`, as that is the top-level `python3` and not the python of the package set. Actually, you already use `python.version` so this parameter is now unused.